crates/re_viewer/src/ui/recordings_panel.rs

```diff
@@ -80,7 +80,7 @@ fn recording_list_ui(ctx: &mut ViewerContext<'_>, ui: &mut egui::Ui) -> bool {
         } else {
             ctx.re_ui.list_item(app_id).active(false).show_collapsing(
                 ui,
-                ui.id().with(app_id),
+                ui.make_persistent_id(ui.id().with(app_id)),
```

Comment:
This is putting a hat on a hat

![id-docs](https://github.com/rerun-io/rerun/assets/1148717/22ab3a88-5d78-4c9a-bd29-ca0df77d4d91)